orion/src/misc/technical.md

```diff
@@ -20,9 +20,16 @@ orion://search?q=query to perform search
 
 Location of Open Tabs
 ```
-~/Library/Application\ Support/Orion/Defaults/browser_state.plist 
+~/Library/Application\ Support/Orion/Defaults/named_windows.plist (Seems to contain only the tabs of the named window with lowest ID integer)
```

Comment:
Lowest ID is an assumption. I've tried reordering and various juggling of tabs, but nothing I've done has made any change from the named_windows.plist only containing the window with ID 0 on my instance.